filament/src/PostProcessManager.cpp

```diff
@@ -350,7 +350,7 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::structure(FrameGraph& fg,
                     mi->setParameter("level", uint32_t(level));
                     commitAndRender(out, material, driver);
                 }
-                driver.setMinMaxLevels(in, 0, levelCount);
+                driver.setMinMaxLevels(in, 0, levelCount - 1);
```

Comment:
out of curiosity, does it matter? I thought the default value was like 1000 on gl?

Reply:
Yeah, to pass validation Vulkan cannot create an ImageView with a SubresourceRange that exceeds the bounds of the image.  I should perhaps make the backend auto-clamp.  Either way this codeline was wrong.

Reply:
autoclamp seems like a good idea, for when we'll make these api public. We want to avoid errors in the backend.

Reply:
Done.